docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -42,80 +42,94 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
   * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
   * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
   * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
-     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.9 are recommended. 
+     * Note that OpenVINO is gradually phasing out support for Python 3.6. Python 3.7 - 3.9 are recommended. 
 
 @endsphinxdirective
 
 ## Installing OpenVINO Runtime
 
+### <a name="install-openvino"></a>Step 1: Download and Install the OpenVINO Core Components
+
+First, open a terminal using Ctrl + Alt + T. Create a folder for OpenVINO and move into it by issuing the following commands. If the `/opt/intel` folder already exists, skip the `mkdir` command.
+
+```sh
+sudo mkdir /opt/intel
+cd /opt/intel
+```
+
+> **NOTE**: The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/Intel`. You may use a different path if desired.
```

Comment:
```suggestion
> **NOTE**: The `/opt/intel` path is the recommended folder path for administrators or root users. If you prefer to install OpenVINO in regular userspace, the recommended path is `/home/<USER>/intel`. You may use a different path if desired.
```
For consistency with the installer, in 2022.1 the path in $HOME was with a lowercase i.